biorbd_optim/penalty.py

```diff
@@ -251,6 +279,7 @@ def _parameter_modifier(penalty_function, parameters):
             or penalty_function == PenaltyType.MINIMIZE_ALL_CONTROLS
             or penalty_function == PenaltyType.ALIGN_SEGMENT_WITH_CUSTOM_RT
             or penalty_function == PenaltyType.ALIGN_MARKER_WITH_SEGMENT_AXIS
+            or penalty_function == PenaltyType.TRACK_CONTACT_FORCES
```

Comment:
Please move this right after TRACK_ALL_CONTROLS

Reply:
done

Reply:
done